ironfish/src/metrics/ewmaAverage.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Assert } from '../assert'
+
+/**
+ * Utility to compute the exponential weighted moving average
+ *
+ * inspired by https://github.com/shaka-project/shaka-player/blob/main/lib/abr/ewma.js
```

Comment:
Is this code have a compatible license? We need to check that as well. If so, link to the git commit with the hash specifically rather than master, since master can change.

Reply:
Apache license, i've replaced with the git commit